examples/point-of-sale/package.json

```diff
@@ -20,47 +20,46 @@
     },
     "dependencies": {
         "@solana/pay": "^0.2.4",
-        "@solana/spl-token": "^0.3.5",
-        "@solana/wallet-adapter-base": "^0.9.16",
-        "@solana/wallet-adapter-react": "^0.15.18",
-        "@solana/wallet-adapter-react-ui": "^0.9.16",
-        "@solana/wallet-adapter-wallets": "^0.18.9",
-        "@solana/web3.js": "^1.56.0",
-        "bignumber.js": "^9.1.0",
-        "bs58": "^4.0.1",
-        "cache-manager": "^4.1.0",
+        "@solana/spl-token": "^0.3.6",
+        "@solana/wallet-adapter-base": "^0.9.19",
+        "@solana/wallet-adapter-react": "^0.15.25",
+        "@solana/wallet-adapter-react-ui": "^0.9.23",
+        "@solana/wallet-adapter-wallets": "^0.19.7",
+        "@solana/web3.js": "^1.69.0",
+        "bignumber.js": "^9.1.1",
+        "bs58": "^5.0.0",
+        "cache-manager": "^5.1.4",
```

Comment:
What breaking changes does this introduce?

Reply:
What do you mean ? 
Is it breaking something is the app ? No.
Is it worth the risk to update those packages ? Yes, removed a lot of warning while compiling.